AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/README.md

```diff
@@ -99,15 +98,31 @@ To open the Jupyter notebook on your local server:
 
 ### Run the Sample in the Intel® DevCloud for oneAPI JupyterLab
 
-1. Open the following link in your browser: https://jupyter.oneapi.devcloud.intel.com/
-
-2. In the Notebook Dashboard, navigate to the ``IntelModin_GettingStarted.ipynb`` file and open it.
-
-   **Important:** You must edit the cell that imports modin to enable the Dask or OmniSci backend engine. The Ray backend cannot be used on Intel DevCloud at this time. For more information, visit the [Intel® Distribution of Modin Getting Started Guide](https://www.intel.com/content/www/us/en/developer/articles/technical/intel-distribution-of-modin-getting-started-guide.html).
-
-3. To change the kernel, click **Kernel** > **Change kernel** > **usr_modin**.
-
-4. Run the sample code and read the explanations in the notebook.
+1. If you do not already have an account, request an Intel® DevCloud account at [Create an Intel® DevCloud Account](https://www.intel.com/content/www/us/en/forms/idz/devcloud-registration.html).
+   
+2. Open the following link in your browser: https://devcloud.intel.com/oneapi/get_started/, locate the **Connect with Jupyter Lab\*** section (near the bottom). 
+   
+3. Click **Sign in to Connect** button. (If you are already signed in, the link should say **Launch JupyterLab\***.)
+   
+4. If the samples are not already present in your Intel® DevCloud account, download them.
+   - From JupyterLab, select **File > New > Terminal**.
+   - In the terminal, clone the samples from GitHub: 
+      ```
+      git clone https://github.com/oneapi-src/oneAPI-samples.git
+      ```
+
+5. Setup environment in the terminal:
+   - source oneAPI conda environment
+      ```
+      source /opt/intel/oneapi/setvars.sh --force
+      ```
+   - Refer to [Environment Setup](#jump0) to setup environment 
+  
+6. In the JupyterLab, navigate to the ``IntelModin_GettingStarted.ipynb`` file and open it.
+
+7. To change the kernel, click **Kernel** > **Change kernel** > **usr_modin**.
```

Comment:
you might need to register your new conda kernel as one of the jupyter notebook first before you switches the notebook kernel to it.

Reply:
> you might need to register your new conda kernel as one of the jupyter notebook first before you switches the notebook kernel to it.

Hi Louie, it is described in Environment Setup session which will be done before changing kernel.
4. Create a new kernel for Jupyter Notebook based on your activated conda environment.
   ``` bash
   conda install ipykernel
   python -m ipykernel install --user --name usr_modin
   ```

Reply:
thanks for reminding.